packaging/agents/copy_packages.py

```diff
@@ -19,7 +19,7 @@ def splitext(filename):
 
 
 def normalize_agent_name(filename):
-    return filename.split("_", 1)[0].lower()
+    return filename.rpartition("_")[0].lower()
```

Comment:
rsplit('_', 1) would also work- any reason you preferred to shift to partition? (longer returned value though negligibly for what we're doing)